include/SDL3/SDL_time.h

```diff
@@ -0,0 +1,244 @@
+    int utc_offset;            /**< Seconds east of UTC */
+} SDL_DateTime;
+
+typedef enum SDL_DAY_OF_WEEK
+{
+    SDL_DAY_OF_WEEK_SUNDAY = 0,
+    SDL_DAY_OF_WEEK_MONDAY = 1,
+    SDL_DAY_OF_WEEK_TUESDAY = 2,
+    SDL_DAY_OF_WEEK_WEDNESDAY = 3,
+    SDL_DAY_OF_WEEK_THURSDAY = 4,
+    SDL_DAY_OF_WEEK_FRIDAY = 5,
+    SDL_DAY_OF_WEEK_SATURDAY = 6
+} SDL_DAY_OF_WEEK;
+
+/**
+ *
+ * The preferred date format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_DATE_FORMAT
+{
+    SDL_DATE_FORMAT_YYYYMMDD = 0, /**< year/month/day */
+    SDL_DATE_FORMAT_DDMMYYYY = 1, /**< day/month/year */
+    SDL_DATE_FORMAT_MMDDYYYY = 2, /**< month/day/year */
+} SDL_DATE_FORMAT;
+
+/**
+ *
+ * The preferred time format of the current system locale.
+ *
+ * \sa SDL_GetDateTimeLocalePreferences
+ */
+typedef enum SDL_TIME_FORMAT
+{
+    SDL_TIME_FORMAT_24HR = 0, /**< 24 hour time */
+    SDL_TIME_FORMAT_12HR = 1, /**< 12 hour time */
+} SDL_TIME_FORMAT;
+
+/**
+ *  Global date/time properties
+ *
+ *  - `SDL_PROP_GLOBAL_SYSTEM_DATE_FORMAT_NUMBER`: the index of the SDL_DATE_FORMAT enum entry with the
+ *    preferred date display format for the system locale.
+ *  - `SDL_PROP_GLOBAL_SYSTEM_TIME_FORMAT_NUMBER`: the index of the SDL_TIME_FORMAT enum entry with the
+ *    preferred time display format for the current locale.
+ */
+#define SDL_PROP_GLOBAL_SYSTEM_DATE_FORMAT_NUMBER "time.date_format"
+#define SDL_PROP_GLOBAL_SYSTEM_TIME_FORMAT_NUMBER "time.time_format"
+
+/**
+ *
+ * Gets the current value of the system realtime clock in nanoseconds since Jan 1, 1970 UTC.
+ *
+ * \param ticks the SDL_Time to hold the returned tick count
+ * \returns 0 on success or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0 
+ */
+extern DECLSPEC int SDLCALL SDL_GetRealtimeClockTicks(SDL_Time *ticks);
+
+/**
+ *
+ * Converts a given SDL_TimeSpec since the epoch to a calendar time, expressed in Universal
+ * Coordinated Time (UTC), in the SDL_DateTime format.
+ *
+ * \param ticks the SDL_Time to be converted
+ * \param dt the resulting SDL_DateTime
+ * \returns 0 on success or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0
+ */
+extern DECLSPEC int SDLCALL SDL_TimeToUTCDateTime(SDL_Time ticks, SDL_DateTime *dt);
+
+/**
+ *
+ * Converts a given SDL_TimeSpec since the epoch to a calendar time, expressed in local time,
+ * in the SDL_DateTime format.
+ *
+ * \param ticks the SDL_Time to be converted
+ * \param dt the resulting SDL_DateTime
+ * \returns 0 on success or -1 on error; call SDL_GetError() for more information.
+ *
+ * \since This function is available since SDL 3.0.0
+ */
+extern DECLSPEC int SDLCALL SDL_TimeToLocalDateTime(SDL_Time ticks, SDL_DateTime *dt);
+
+/**
+ *
+ * Converts a calendar time to seconds and nanoseconds since the epoch.
```

Comment:
"Converts a calendar time to an SDL_Time value."

Reply:
Fixed